docs/guides/custom-functions.md

```diff
@@ -183,7 +183,7 @@ export default (obj) => {
 };
 ```
 
-The following code won't work properly either:
+Require calls will work only in Node.js, therefore it's strongly discouraged to use them unless you don't plan to distribute your ruleset among other users:
 
  ```js
 const foo = require('./foo'); // require is not available (see note below)
```

Comment:
@P0lip In the context of that PR, that comment "// require is not available (see note below)" becomes a little bit confusing

Reply:
```suggestion
const foo = require('./foo');
```